CONTRIBUTING.md

```diff
@@ -30,52 +52,96 @@ For more information specific to how a quarto site works, and for examples of wh
+change the display of code blocks using predefined themes. This will
+change the appearance of all code snippets across the entire Analysts'
+guide. The current theme is set to "printing" but there is a list of
+other available themes [on the Quarto
+website](https://quarto.org/docs/output-formats/html-code.html#highlighting).
+
+Some things to be aware of before you make changes:
+
+-   Some of the themes are adaptive, meaning that if you change the site
+    view from dark mode to light mode then the theme will also change
+    accordingly.
+
+-   The appearance of code snippets will only change if the language
+    (e.g. R, Python) is defined at the start of the snippet (e.g.
+    ```` ``` {r connection_example, eval=FALSE} ````). Including
+    `eval=FALSE` stops your code snippet from actually running. If no
+    language is defined, the snippet appearance will need to be modified
+    manually and this can cause some issues.
 
 ### Issues
 
 #### Create a new issue
 
-If you spot a problem with the site, [search if an issue already exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments). If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/dfe-analytical-services/analysts-guide/issues/new/choose).
+If you spot a problem with the site, [search if an issue already
+exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments).
+If a related issue doesn't exist, you can open a new issue using a
+relevant [issue
+form](https://github.com/dfe-analytical-services/analysts-guide/issues/new/choose).
 
 #### Solve an issue
 
-Scan through our [existing issues](https://github.com/dfe-analytical-services/analysts-guide/issues) to find one that interests you. If you find an issue to work on, you are welcome to open a PR with a fix.
+Scan through our [existing
+issues](https://github.com/dfe-analytical-services/analysts-guide/issues)
+to find one that interests you. If you find an issue to work on, you are
+welcome to open a PR with a fix.
 
 ### Make Changes
 
-Key things to remember when making or proposing changes:
-- Where guidance already exists elsewhere it should be linked to rather than duplicated
-- When adding images, save them in the /images folder
-- When adding files for download, save them in the /resources folder
-- Don't commit rendered .html files for pages
-- If your changes remove or edit a pre-existing anchor link, consider how that will be redirected for users who may have bookmarked it
+Key things to remember when making or proposing changes: - Where
+guidance already exists elsewhere it should be linked to rather than
+duplicated - When adding images, save them in the /images folder - When
```

Comment:
Think we've lost the bullet point formatting, was this intentional? ([your branch](https://github.com/dfe-analytical-services/analysts-guide/blob/rap-page-additions/CONTRIBUTING.md) is on the left and the [main branch preview](https://github.com/dfe-analytical-services/analysts-guide/blob/main/CONTRIBUTING.md) is on the right in the screenshot below)

![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/b43c9a01-372c-4bcb-9377-5c1ffdd1b073)